egs/multi_en/s5/local/remove_dup_utts.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
this script local/remove_dup_utts.sh should be deleted as it now lives in utils/data/.